tools/fuchsia/devshell/build_and_copy_to_fuchsia.sh

```diff
@@ -0,0 +1,159 @@
+engine-info "Building ${fuchsia_out_dir_name}..."
+${ninja_cmd} -C "${fuchsia_out_dir}" flutter/shell/platform/fuchsia
+
+engine-info "Making Fuchsia's Flutter prebuilts writable..."
+chmod -R +w "$FUCHSIA_DIR"/prebuilt/third_party/flutter
+
+engine-info "Copying the patched SDK (dart:ui, dart:zircon, dart:fuchsia) to Fuchsia..."
+cp -ra "${fuchsia_out_dir}"/flutter_runner_patched_sdk/* "$FUCHSIA_DIR"/prebuilt/third_party/flutter/"${fuchsia_cpu}"/release/aot/flutter_runner_patched_sdk/
+
+engine-info "Registering debug symbols..."
+"$ENGINE_DIR"/fuchsia/sdk/linux/tools/x64/symbol-index add "${fuchsia_out_dir}"/.build-id "${fuchsia_out_dir}"
+
+if [[ "${runtime_mode}" == release ]]
+then
+  flutter_runner_pkg="flutter_jit_product_runner-0.far"
+  engine-info "Copying the Flutter JIT product runner (${flutter_runner_pkg}) to Fuchsia..."
+  cp "${fuchsia_out_dir}"/"${flutter_runner_pkg}" "$FUCHSIA_DIR"/prebuilt/third_party/flutter/"${fuchsia_cpu}"/"${runtime_mode}"/jit/"${flutter_runner_pkg}"
+
+  flutter_runner_pkg="flutter_aot_product_runner-0.far"
+  engine-info "Copying the Flutter AOT product runner (${flutter_runner_pkg}) to Fuchsia..."
+  cp "${fuchsia_out_dir}"/"${flutter_runner_pkg}" "$FUCHSIA_DIR"/prebuilt/third_party/flutter/"${fuchsia_cpu}"/"${runtime_mode}"/aot/"${flutter_runner_pkg}"
+
+  dart_runner_pkg="dart_jit_product_runner-0.far"
+  engine-info "Copying the Dart JIT product runner (${dart_runner_pkg}) to Fuchsia..."
+  cp "${fuchsia_out_dir}"/"${dart_runner_pkg}" "$FUCHSIA_DIR"/prebuilt/third_party/flutter/"${fuchsia_cpu}"/"${runtime_mode}"/jit/"${dart_runner_pkg}"
+
+  dart_runner_pkg="dart_aot_product_runner-0.far"
+  engine-info "Copying the Dart AOT product runner (${dart_runner_pkg}) to Fuchsia..."
+  cp "${fuchsia_out_dir}"/"${dart_runner_pkg}" "$FUCHSIA_DIR"/prebuilt/third_party/flutter/"${fuchsia_cpu}"/"${runtime_mode}"/aot/"${dart_runner_pkg}"
+else
+  flutter_runner_pkg="flutter_${compilation_mode}_runner-0.far"
+  engine-info "Copying the Flutter runner (${flutter_runner_pkg}) to Fuchsia..."
+  cp "${fuchsia_out_dir}"/"${flutter_runner_pkg}" "$FUCHSIA_DIR"/prebuilt/third_party/flutter/"${fuchsia_cpu}"/"${runtime_mode}"/"${compilation_mode}"/"${flutter_runner_pkg}"
+
+  dart_runner_pkg="dart_${compilation_mode}_runner-0.far"
+  engine-info "Copying the Dart runner (${dart_runner_pkg}) to Fuchsia..."
+  cp "${fuchsia_out_dir}"/"${dart_runner_pkg}" "$FUCHSIA_DIR"/prebuilt/third_party/flutter/"${fuchsia_cpu}"/"${runtime_mode}"/"${compilation_mode}"/"${dart_runner_pkg}"
+fi
+
+# TODO(akbiggs): Warn the developer when their current
+# Fuchsia configuration (`fx set`) is mismatched with the runner
+# they just deployed.
+
+# TODO(akbiggs): Copy the tests over. I couldn't figure out a glob that grabs all of them.
+
+echo "Done. You can now build Fuchsia with your Flutter Engine changes by running:"
+echo '  cd $FUCHSIA_DIR'
+# TODO(akbiggs): I'm not sure what example to give for arm64.
```

Comment:
Does anyone use `workstation_eng.arm64`?

Reply:
It seems to be broken for me, opened fxb/108361.